test_api.py

```diff
@@ -38,13 +38,20 @@ def test_daemonstatus_ok():
 def test_listprojects_ok():
     response = requests.get(BASE_URL + '/listprojects.json')
     assert_response_ok(response)
-    assert response.json() == { 'status': 'ok', 'projects': AVAIL_PROJECTS }
+
+    json = response.json()
+    assert json['projects'] == AVAIL_PROJECTS
+    assert 'node_name' in json
```

Comment:
What about setting a `node_name` in the example config, then we can also test its value?

We don't have the infrastructure to test config file variations at the moment.

Reply:
Maybe let's make it a ticket out of it?

Reply:
Ok, maybe good indeed to not have this in the example config files.